yarn.lock

```diff
@@ -4009,9 +4009,9 @@ atob@^2.1.2:
   integrity sha512-Wm6ukoaOGJi/73p/cl2GvLjTI5JM1k/O14isD73YML8StrH/7/lRFgmg8nICZgD3bZZvjwCGxtMOD3wWNAu8cg==
 
 ava@^5.1.0:
-  version "5.1.0"
-  resolved "https://registry.yarnpkg.com/ava/-/ava-5.1.0.tgz#98c4aac86a427c9843d3af13f25552b71014bff7"
-  integrity sha512-e5VFrSQ0WBPyZJWRXVrO7RFOizFeNM0t2PORwrPvWtApgkORI6cvGnY3GX1G+lzpd0HjqNx5Jus22AhxVnUMNA==
+  version "5.1.1"
```

Comment:
did anything in particular prompt this? It seems like a considerable amount of churn for reasons that aren't clear to me. The [release notes](https://github.com/avajs/ava/releases/tag/v5.1.1) show only:
> - Fix de-registration of shared workers to ensure AVA exits correctly

Reply:
Changes in dependencies might introduce a bug.